### PR TITLE
made mutation edit button x-large to make it easier to click.

### LINKED ISSUE #1096: Make hit area of mutation (aka command) edit button bigger
**Describe exactly what you would like to see in an upcoming release**

The "edit" button in the command menu has an unreasonably small hit area. It is too easy to submit the command when trying to click the edit button.

image

**Pull requests welcome!**

### PR DESCRIPTION
**Pull request by wxtim**
May close #1096 

**Requirements check-list**
- [x] I have read `CONTRIBUTING.md` and added my name as a Code Contributor.
- [x] Contains logically grouped changes (else tidy your branch by rebase).
- [x] Does not contain off-topic changes (use other PRs for other changes).
- [x] Already covered by existing tests.
- [x] No change log entry required (v small change)
- [x] No documentation update required.

### CODE REVIEW

**Review by MetRonnie — CHANGES_REQUESTED** [review]
Icon is now outsized

image

I have opened https://github.com/wxtim/cylc-ui/pull/4 against this with some improvements

File: src/styles/cylc/_menu.scss [hunk 1]
```diff
@@ -38,4 +38,7 @@
       line-height: 1.35;
     }
   }
+  .pad-button {
+    padding: 30px;
+  }
```
Comment (MetRonnie) [inline src/styles/cylc/_menu.scss:43]: There is no element with the class `pad-button` so this does nothing

Comment (wxtim) [reply src/styles/cylc/_menu.scss:43]: I did not want to commit this file - this was something I played with and discarded.

**Comment by wxtim** [comment]
@MetRonnie  I've removed the `icon` property from the button so it goes back to being square, and I've added an outline so it's completely clear what is and is not the button. I think either of these measures would be sufficient on their own, but in combination...

**Comment by MetRonnie** [comment]
Not sure I like the rectangular button or outline. IMHO draws too much attention for a secondary control.

image

My proposal:

image

**Comment by wxtim** [comment]
> Not sure I like the rectangular button or outline. IMHO draws too much attention for a secondary control.
> 
> image
> 
> My proposal (reverts [f079595](https://github.com/cylc/cylc-ui/commit/f079595e5742a7f6af8540b10fcb7ec0f7c1a95c)):
> 
> image

How about the rectangular button without the edge? 

I don't think anyone will move their mouse over there if they don't want the edit button, and pressing the edit button wrongly is the fail-safe way...

**Comment by hjoliver** [comment]
I think the subtly-shaded round button looks best.  Is @MetRonnie's proposal still bigger/more-clickable than the status quo?

**Comment by wxtim** [comment]
> I think the subtly-shaded round button looks classiest. Is @MetRonnie's proposal bigger/more-clickable than the status quo? [Looking back at the issue, it is bigger - is that not enough in your view @wxtim ?]

Whatever, I took this on as a learning experience, I'm not militant - If the consensus is the round button, I'll do the round button.

**Pull request merged**